tklock.c

```diff
@@ -294,6 +304,81 @@ static void disable_autorelock_policy(void)
 	return;
 }
 
+/**
+ * Enable/disable X11 input events (currently only for touchscreen)
+ *
```

Comment:
Then rename it to generic_x11_event_ts_control() or somesuch. The other (and better option IMO) is to pass the Atom of the desired type to be controlled. Not really needed to be done with this PR though, just keep that in mind.

Reply:
I was expecting to indeed pass an atom (or some internal type) to this function, that's why I named it as such and put that comment there.